extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/base/ModeledDirectoryObjectService.java

```diff
@@ -510,7 +530,7 @@ public void deleteObject(ModeledAuthenticatedUser user, String identifier)
         beforeDelete(user, identifier);
         
         // Delete object
-        getObjectMapper().delete(identifier);
+        getObjectMapper().delete(identifier, environment.getCaseSensitiveUsernames());
```

Comment:
Since this is the base class, I think this should be `getCaseSensitiveIdentifiers()`, not `environment.getCaseSensitiveUsernames()`.

Reply:
Thanks for catching that - I've corrected this in a rebase.